src/grading-settings/assignment-section/utils/enum.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+export const DUPLICATE_ASSIGNMENT_NAME = 'duplicateAssignmentName';
+
+export const ASSIGNMENT_TYPES = {
```

Comment:
The only comment about naming: it can be confusing (since it is exported) to use ASSIGNMENT_TYPES not for assignment types, but for assignment type fields.